### PR TITLE
LPS-199858 UI adjustments in Page Audit warnings

### PR DESCRIPTION
**Pull request by claraizquierdo**
Motivation
=======

Fix left margin in warning description
[Link to story or ticket](https://liferay.atlassian.net/browse/LPS-199858)

Steps to Verify:
----------------
1. In Documents and Media add a big image (>500kb)
2. Go to home page and add an image fragment with the image uploaded in step 1
3. In the image fragment configuration select resolution: auto
4. Publish the page
5. Open the Page Audit and click on the image fragment

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/ItemDetail.js [hunk 1]
```diff
@@ -289,12 +289,10 @@ function Warning({item}) {
 	return (
 		<ClayList.Item className="border-0 c-p-0">
 			<ClayAlert displayType="warning" role="none" variant="feedback">
-				<span className="c-mb-3 d-block text-weight-semi-bold">
-					{item.title}
-				</span>
-
-				<span>{item.description}</span>
+				<span className="text-weight-semi-bold">{item.title}</span>
```
Comment (veroglez) [inline modules/apps/layout/layout-reports-web/src/main/resources/META-INF/resources/js/components/ItemDetail.js:292]: We can use the props `title` of the `ClayAlert` and this way we avoid adding an extra `<span>`

**Comment by veroglez** [comment]
@claraizquierdo I left only a tiny comment 🙂

**Comment by veroglez** [comment]
ci:forward

**Comment by liulukiwi** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/142689

**Comment by liulukiwi** [comment]
ci:close

**Pull request closed without merging**